test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -337,6 +337,19 @@ export default class BaseFormObject {
   }
 
   async selectRadioButtonOptionByValue(fieldName, value) {
+    if (userVariables.v3) {
+      const radioOption = await this.el.find(`input[type="radio"][value="${value}"]`);
+      await this.t.click(radioOption); 
+
+      const label = this.el
+        .find(`input[type="radio"][value="${value}"]`)
+        .parent('span')
+        .parent('label')
+        .textContent;
+
+      return label; 
+    }
```

Comment:
Is there a better way to do this? i.e. not pass a dummy `` value as 1st argument to selectRadioButtonOptionByValue
In v3, we don't have a field name for radio options.
In v2, field name is used like this -
`this.el.find([data-se="${fieldName}"]`);`

In v3, we can use just the value field (2nd param) -
`this.el.find(`input[type="radio"][value="${value}"]`);`

I don't want to create another method that just takes 1 argument, since methods with same names are not allowed according to lint rules.